### ISSUE TITLE
Show a counter somewhere at the bottom of the page

### ISSUE THREAD

**Issue by nikhilchh** [csubhodeep/ride-sharing-app#22]
X rides created in total (*Only display after 1000+)
Y rides per day (start displaying after 100 days)

This will give people an assurance that there are riders and passengers on this platform. It will motivate them to try this platform.

**Comment by csubhodeep** [comment]
Yes I was thinking but then I thought wouldn't be too nerdy? But if you think it would make sense we could do it. Maybe later we also show a small heatmap on the map of CA.

**Comment by nikhilchh** [comment]
yes heatmap is also a good idea.
